src/main/java/com/example/gradecalculator/web/controller/FormController.java

```diff
@@ -1,34 +1,37 @@
 package com.example.gradecalculator.web.controller;
-
-import com.example.gradecalculator.entities.UserType;
-import com.example.gradecalculator.repository.UserTypeRepository;
-import org.apache.catalina.User;
+import com.example.gradecalculator.entities.AllUsers;
+import com.example.gradecalculator.repository.AllUsersRepository;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.autoconfigure.security.SecurityProperties;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.ResponseBody;
 
 @Controller
+@SpringBootApplication
 public class FormController {
 
-    private final UserTypeRepository userTypeRepository;
+    private final AllUsersRepository userRepository;
 
     @Autowired
-    public FormController(UserTypeRepository userTypeRepository) {
-        this.userTypeRepository = userTypeRepository;
+    public FormController(AllUsersRepository userRepository) {
+        this.userRepository = userRepository;
     }
 
     @PostMapping("/submit")
     @ResponseBody
-    public String handleSubmit(@RequestParam String name, @RequestParam String email, @RequestParam String message, @RequestParam Long userTypeId) {
-        UserType userType = userTypeRepository.findById(userTypeId).orElse(null);
-
-        // ToDo insert user
-        // ConcreteUser user = new ConcreteUser(name, email, message, userType);
-        // userTypeRepository.save(user);
+    public String handleSubmit(@RequestParam String name, @RequestParam String email, @RequestParam String message) {
+        AllUsers user = new AllUsers(name, email, message);
+        userRepository.save(user);
+        System.out.println("Name: " + name);
+        System.out.println("E-Mail: " + email);
+        System.out.println("Nachricht: " + message);
+        return "Daten wurden im Terminal angezeigt.";
+    }
 
-        return "Data has been saved in the backend.";
+    public static void main(String[] args) {
```

Comment:
why is there a main method in a controller?

Reply:
fixed